docs/docs/configuration/live.md

```diff
@@ -17,7 +17,7 @@ Live view options can be selected while viewing the live stream. The options are
 
 ### WebRTC extra configuration:
 
-WebRTC works by creating a WebSocket connection on port `8555`. However, it requires additional configuration:
+WebRTC works through by creating a TCP or UDP connection on port `8555`. However, it requires additional configuration:
```

Comment:
adding `through` doesn't make sense to me?

Reply:
```suggestion
WebRTC works by creating a TCP or UDP connection on port `8555`. However, it requires additional configuration:
```